registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflowmanager/verticle/WorkflowInternalActionVerticle.java

```diff
@@ -87,6 +88,9 @@
 
 	@Value("${mosip.regproc.workflow-manager.internal.action.eventbus.port}")
 	private String eventBusPort;
+	
+    @Value("${mosip.regproc.workflow-manager.internal.action.max.allowed.iteration}")

```

Comment:
max.allowed.iteration - change to max-allowed-iteration

Reply:
fixed